### PR TITLE
Added ability to embed the timer as a widget on an external website

### PR DESCRIPTION
**Pull request by LeonGeldsch**
I added a new script that allows the timer to be embedded on other websites. I also added a new HTML page and updated the README to give examples on how to integrate it.

### CODE REVIEW

File: dist/embed.js [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
...t:500px;--timer-width:500px}@media (max-width:500px){:root{--timer-height:100vw;--timer-width:100vw}}#base-timer{position:relative;margin:auto;font-family:"Courier Prime",monospace;color:#fff}#base-timer .circle{fill:none;stroke:none}#base-timer .path-elapsed{stroke-width:5px;stroke:white}.inside-timer{position:absolute;top:0;grid-template-rows:1fr 1fr 1fr}#current-interval{grid-column:2/3;font-size:35px;text-align:center;margin:40% auto auto}#time-left-display{font-size:48px;grid-column:2/3;margin:auto}.timer-controls{grid-column:2/3}#path-remaining{stroke-width:5px;stroke-linecap:round;transform:rotate(90deg);transform-origin:center;transition:.5s linear;stroke:currentColor;color:var(--path-time-color)}.timer-controls button{box-sizing:border-box;margin:0 auto;padding:1px;border:2px solid #fff;border-radius:20%;width:40px;height:40px;background:#172a3a;color:#fff;cursor:pointer;transition:background 250ms ease-in-out,transform 150ms;-webkit-appearance:none;-moz-appearance:none}@media (min-width:500px){.timer-controls button{margin:40% auto auto}}.timer-controls button::-moz-focus-inner{border:none}.timer-controls button:hover{background:#0053ba}svg{transform:scaleX(-1)}</style>';
```
Comment (Districtfine) [inline dist/embed.js]: See if you can add some formatting for this by using template literals:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

File: src/scripts/embed.js
Comment (Districtfine) [inline src/scripts/embed.js:1]: Could you explain to me why this file is needed? It's nearly identical to dist/embed.js

File: embed-example.html
Comment (Districtfine) [inline embed-example.html:1]: Not needed, I think the explanation in the readme was sufficent

File: dist/embed.js [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+  if (momentScriptReady && countdownScriptReady) start();
+};
+countdownScript.src =
+  "https://cdnjs.cloudflare.com/ajax/libs/countdown/2.6.0/countdown.min.js";
+document.head.appendChild(countdownScript);
+
+function start() {
+  createEmbedElements();
+
+  function selectElement(id, valueToSelect) {
```
Comment (Districtfine) [inline dist/embed.js]: Looks like from this point on the code is almost the same as scripts/index.js is there a way that we can have index.js reference code in here? Best to keep things as DRY as possible

**Review by Districtfine — CHANGES_REQUESTED** [review]
Thanks for the PR! Mostly looks good to me, my biggest concern is the repeated content, would be good if there was a way to deduplicate

**Comment by LeonGeldsch** [comment]
Thank you for the suggestions. I have simplified `embed.js` to import all of the scripts externally. I also reformatted the embedded HTML to be more readable.

**Comment by LeonGeldsch** [comment]
I've also deleted the duplicate `embed.js` in the `dist` folder. The reason I created it in the first place is because I saw you were using babel to process your source files before uploading them. I tried to do the same (which is why I also updated the "build" command in the `package.json`) but for some reason I kept getting an error after parsing the code with babel, so I just changed it to copy.

I've deleted the duplicate file for now.

**Comment by Districtfine** [comment]
Thanks! Looks good

**Pull request merged**